README.md

```diff
@@ -48,11 +48,12 @@ For perplexity comparison, you can turn to [here](https://github.com/qwopqwop200
 
 AutoGPTQ is available on Linux and Windows only. You can install the latest stable release of AutoGPTQ from pip with pre-built wheels:
 
-| CUDA/ROCm version | Installation                                                                                      | Built against PyTorch |
+| Platform version | Installation                                                                                      | Built against PyTorch |
 |-------------------|---------------------------------------------------------------------------------------------------|-----------------------|
 | CUDA 11.8         | `pip install auto-gptq --no-build-isolation --extra-index-url https://huggingface.github.io/autogptq-index/whl/cu118/`   | 2.2.1+cu118           |
 | CUDA 12.1         | `pip install auto-gptq --no-build-isolation`                                                                            | 2.2.1+cu121           |
-| ROCm 5.7          | `pip install auto-gptq --no-build-isolation --extra-index-url https://huggingface.github.io/autogptq-index/whl/rocm573/` | 2.2.1+rocm5.7               |
+| ROCm 5.7          | `pip install auto-gptq --no-build-isolation --extra-index-url https://huggingface.github.io/autogptq-index/whl/rocm573/` | 2.2.1+rocm5.7
+| Intel Gaudi HPU 1.17          | `BUILD_CUDA_EXT=0 pip install auto-gptq --no-build-isolation` | [2.3.0+Intel Gaudi 1.17](https://docs.habana.ai/en/latest/Installation_Guide/)               |
 
 AutoGPTQ can be installed with the Triton dependency with `pip install auto-gptq[triton] --no-build-isolation` in order to be able to use the Triton backend (currently only supports linux, no 3-bits quantization).
```

Comment:
AutoGPTQ can be installed with the Triton dependency using `pip install auto-gptq[triton] --no-build-isolation`, allowing users to access the Triton backend (currently only supports Linux, no 3-bits quantization).

Reply:
This is not my change, it's from upstream